deploy/Chart/templates/rp/deployment.yaml

```diff
@@ -83,6 +87,15 @@ spec:
         - name: config-volume
           configMap:
             name: applications-rp-config
+        {{- if eq .Values.global.aws.irsa.enabled true }}
+        - name: aws-iam-token
```

Comment:
```suggestion
        - name: aws-irsa-token
```

Reply:
as discussed offline keeping it `aws-iam-token`